app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: "noreply@example.com"
```

Comment:
thêm biến môi trường cho email em nhé

Reply:
thêm biến môi trường vào bị lỗi SMTP From address may not be blank: nil ạ T_T